app/javascript/components/registrations/Login.jsx

```diff
@@ -66,52 +64,51 @@ class Login extends Component {
             <Header as="h2" color="teal" textAlign="center">
               Log In
             </Header>
-            {this.state.error
-            ? (
+            {this.state.error ? (
```

Comment:
can change to {this.state.error && <Compoenent/>} instead of {this.state.error ? <Component/> : <EmptyComponent/> this will render the component if is `this.state.error == true`